packages/ember-toucan-core/src/components/form/fields/checkbox-group.hbs

```diff
@@ -2,7 +2,7 @@
   <fieldset
     aria-describedby="{{if @error field.errorId}} {{if @hint field.hintId}}"
     disabled={{@isDisabled}}
-    {{! attribute aria-invalid is not supported by the element fieldset with the implicit role of group }}
+    data-root-field={{if @rootTestSelector @rootTestSelector}}
```

Comment:
I think this got lost in the migration off of `<Fieldset`, but it's back!

Reply:
My mistake on this one...I'll look into adding a missing test for this